api/routes.py

```diff
@@ -123,17 +129,26 @@ async def get_all_concepts():
 
 
 @app.put("/concepts/{id}", tags=["concepts"])
-async def create_or_update_concept(id: str, terminology_id: str, name: str):
```

Comment:
Unless the request param name matches the param name, this should not work afaik

Test for e.g.` /concepts/some_id `

Reply:
You are right, I missed it. Fixed in the next commit.